drivers/aks/aks_driver.go

```diff
@@ -905,6 +905,14 @@ func logClusterConfig(config containerservice.ManagedCluster) {
 	}
 }
 
+func (d *Driver) ETCDSave(ctx context.Context, clusterInfo *types.ClusterInfo, opts *types.DriverOptions, snapshotName string) error {
+	return fmt.Errorf("ETCD backup operations are not implemented by this Driver")
+}
+
+func (d *Driver) ETCDRestore(ctx context.Context, clusterInfo *types.ClusterInfo, opts *types.DriverOptions, snapshotName string) error {
+	return fmt.Errorf("ETCD backup operations are not implemented by this Driver")
```

Comment:
can we change `this Driver` to the actual driver name? assuming that some equivalent to `getName` is present in the driver interface